os/arch/arm/src/s5j/s5j_mct.c

```diff
@@ -130,9 +132,56 @@ static void s5j_mct_wait_wstat(uint32_t base, uint32_t bitmask)
 	putreg32(bitmask, reg);
 }
 
+/****************************************************************************
+ * Name: s5j_mct_global_get_comp_cnt
+ *
+ * Description:
+ *   Return current counter value of comparator
```

Comment:
Can you add info about input parameters , any assumptions made and return type . etc to all new APIs?

Reply:
done